generated/provider_metadata.json

```diff
@@ -6857,10 +6857,16 @@
             "date_released": "2023-12-12T07:17:13Z"
         },
         "4.3.1": {
-            "associated_airflow_version": "2.8.1",
+            "associated_airflow_version": "2.8.2",
             "date_released": "2024-01-27T18:56:22Z"
         }
     },
+    "teradata": {
+        "2.0.0": {
+            "associated_airflow_version": "2.0.0",
```

Comment:
mmm this one is odd

Reply:
Indeed :). Because it's not been yet used in any of the Airflow released. We should likely fix the script to put "latest" airflow here, which will get bumped next time after we release 2.8.3 or 2.9.0 - depends where it will be included - but for now it's better to manually add it.

Reply:
BTW. I've actually found out that there is another case that leads to 2.0.0 (look at weaviate 1.0.0). Fix is coming soon

Reply:
Cool! thanks!

Reply:
The https://github.com/apache/airflow/pull/37741 contains much improved provider metadata generaion fixing all the problems found (and improving the speed by adding parallelism)

Reply:
This one can be closed when #37741 is merged - I included metadata refreshed in the PR.